### PR TITLE
Release 0.17.0-prerelease Improvements

### PR DESCRIPTION
**Pull request by SteveMutungi254**
Release 0.17.0-prerelease Improvements

### CODE REVIEW

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaUser.md [hunk 1]
```diff
@@ -175,6 +175,139 @@ New User           cccccccc-2222-3333-4444-dddddddddddd      NewUser@tenant.com
 
 This example demonstrates how to retrieve all users with disabled accounts.
 
+### Example 8: List users based in India
+
+```powershell
+Connect-Entra -Scopes 'User.Read.All'
+$usersInIndia = Get-EntraUser -Filter "Country eq 'India'"
```
Comment (alexandair) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaUser.md:182]: Shouldn't this be `Get-EntraBetaUser`?

**Review by alexandair — CHANGES_REQUESTED** [review]

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaUser.md (hunk 1 above)
Comment (SteveMutungi254) [reply module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaUser.md:182]: Great catch! Thank you.

**Review by emmanuel-karanja — APPROVED** [review]
LGTM

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraServicePrincipal.md [hunk 2]
```diff
@@ -220,6 +220,37 @@ Global secure access app     00001111-aaaa-2222-bbbb-3333cccc4444 33334444-dddd-
...
+```powershell
+Connect-Entra -Scopes 'Application.Read.All'
+$servicePrincipal = Get-EntraServicePrincipal -Filter "PreferredSingleSignOnMode eq 'saml'"
+$servicePrincipal | Select-Object Id, DisplayName, AccountEnabled, AppId, PreferredSingleSignOnMode, AppRoleAssignmentRequired, SignInAudience, NotificationEmailAddresses, PreferredTokenSigningKeyEndDateTime, PreferredTokenSigningKeyValid, ReplyUrls,LoginUrl, LogoutUrl | Format-Table -AutoSize
+```
+
+```Output
+Id                                   DisplayName                           AccountEnabled AppId                                PreferredSingleSignOnMode AppRoleAssignmentRequired SignInAudience NotificationEmailAddresses
+--                                   -----------                           -------------- -----                                ------------------------- ------------------------- -------------- --------------
+00001111-aaaa-2222-bbbb-3333cccc4444 SAML App                             True            33334444-dddd-5555-eeee-6666ffff7777 saml                              True                    AzureADMyOrg   {admin@Contos}
```
Comment (KenitoInc) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraServicePrincipal.md:249]:
```suggestion
00001111-aaaa-2222-bbbb-3333cccc4444 SAML App                             True            33334444-dddd-5555-eeee-6666ffff7777 saml                              True                    AzureADMyOrg   {admin@Contoso}
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaServicePrincipal.md [hunk 3]
```diff
@@ -220,6 +220,37 @@ Global secure access app     00001111-aaaa-2222-bbbb-3333cccc4444 33334444-dddd-
 
 This example demonstrates how to retrieve all Global secure access apps.
 
+### Example 11: List all applications without user assignment
+
+```powershell
+Connect-Entra -Scopes 'Application.Read.All'
+Get-EntraBetaServicePrincipal -All | Where-Object {$_.appRoleAssignmentRequired -ne 'true'}
```
[the remark this answers is not in the document]
Comment (SteveMutungi254) [reply module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaServicePrincipal.md:227]: Synched offline. I am proposing we leave it as per the original proposal.

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaUser.md [hunk 4]
```diff
@@ -175,6 +175,139 @@ New User           cccccccc-2222-3333-4444-dddddddddddd      NewUser@tenant.com
...
+cccccccc-2222-3333-4444-dddddddddddd  New User       NewUser@tenant.com
+bbbbbbbb-1111-2222-3333-cccccccccccc  Sawyer Miller  SawyerM@contoso.com
+```
+
+This example demonstrates how to retrieve users without managers.
+
+### Example 13: List failed sign-ins for a user
+
+```powershell
+Connect-Entra -Scopes 'AuditLog.Read.All','Directory.Read.All'
```
[the remark this answers is not in the document]
Comment (SteveMutungi254) [reply module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaUser.md:288]: That is the right permission:

alt text

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaUser.md [hunk 5]
```diff
@@ -175,6 +175,139 @@ New User           cccccccc-2222-3333-4444-dddddddddddd      NewUser@tenant.com
...
+```powershell
+Connect-Entra -Scopes 'User.Read.All'
+$usersInCanada = Get-EntraBetaUser -Filter "Country eq 'Canada'"
+$usersInCanada | Select-Object Id, DisplayName, UserPrincipalName, OfficeLocation, Country | Format-Table -AutoSize
+```
+
+```Output
+Id                                   DisplayName   UserPrincipalName         OfficeLocation   Country
+--                                   -----------   -----------------         --------------   -------
+cccccccc-2222-3333-4444-dddddddddddd  New User     NewUser@tenant.com        23/2102          Canada
```
[the remark this answers is not in the document]
Comment (SteveMutungi254) [reply module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaUser.md:189]: Corrected. Thank you.

**Review by KenitoInc — APPROVED** [review]

**Pull request merged**